README.md

```diff
@@ -13,9 +13,44 @@ Build and install the provider so that Terraform can use it:
 make install
 ```
 
+## Test
+
+### Unit tests
+
+Unit tests run the provider against a `localhost_apiserver` that serves data read from `/synthetics/test-data.json`
```

Comment:
In this repo we use `test-api-server` instead of `localhost_apiserver` name - take a look at _tools/run_test_api_server.sh_. Another good name for it is fake API server, because it fakes a Kentik API server. I think that localhost_apiserver is a bad name and we should avoid that - `localhost` is unimportant implementation detail.
Please rename within the file.

Reply:
Addressed: 73e1cd8b8c616b3e629ab8ddb25ad0a35ba04fc2